src/styles/antd-overrides/modal.scss

```diff
@@ -29,7 +29,7 @@
 
 .ant-modal-wrap,
 .ant-modal-mask {
-  z-index: 20 !important;
+  z-index: 19 !important;
```

Comment:
Wallet connect z-index is 20 so made it 1 less, to minimise risk of anything else appearing above the ant modal